scripts/travis-publish.sh

```diff
@@ -20,18 +20,24 @@ export publish_cmd="publishLocal"
 if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" && $(cat version.sbt) =~ "-SNAPSHOT" ]]; then
   export publish_cmd="publish gitSnapshots publish"
   # temporarily disable to stabilize travis
-  #if [[ $TRAVIS_SCALA_VERSION = "2.11.8" ]]; then
-  #  export publish_cmd="$publish_cmd ghpagesPushSite"
+  #if [[ $TRAVIS_SCALA_VERSION = "2.11.11" ]]; then
```

Comment:
Kind of tangential to this PR, and it looks like the code is commented out anyway, but we may want to change this to something like `if [[ $TRAVIS_SCALA_VERSION =~ ^2\.11\. ]]`. My knowledge on bash regular expressions is a bit foggy, so this may not be quite right.